README.md

```diff
@@ -90,23 +90,26 @@ To test WUPS, you need to turn on a few REDCap features it interacts with.  You
     update redcap_config set value="1" where field_name = "suspend_users_inactive_send_email";
     update redcap_config set value="30" where field_name = "suspend_users_inactive_days";
 
-As this tool sends emails, make sure the field that will be used for the **Sender Email** address is configured correctly in your module configuration.
+This module has to be configured before you can do anything with it. The instructions that follow assume the module has been configured as described in _Email Configuration Example_ above. The _Days Before Suspension_ setting of `10, 12, 20` is especially important. As this tool sends emails, also make sure the field that will be used for the **Sender Email** address is configured correctly in your module configuration.
 
-You'll also need some test users.  To revise the set of test users `alice`, `bob`, `dan`, and `carol` to receive messages based on the above configuration, change their `user_lastlogin` and `user_lastactivity` dates as follows:
+You'll need some test users. Assuming you have a set of test users `alice`, `bob`, `dan`, and `carol`, you can configure them to receive alerts _today_ by adjusting their `user_lastlogin` and `user_lastactivity` dates as follows:
```

Comment:
```suggestion
You'll need some test users. Assuming you have a set of test users `alice`, `bob`, `carol`, and `dan` you can configure them to receive alerts _today_ by adjusting their `user_lastlogin` and `user_lastactivity` dates as follows:
```
Maybe not really important but the lack of alphabetical order here was nagging my inner pedant.